lms/views/api/canvas/files.py

```diff
@@ -33,16 +41,16 @@ def via_url(self):
         application_instance = self.request.find_service(
             name="application_instance"
         ).get()
-
         assignment = self.request.find_service(name="assignment").get(
             application_instance.tool_consumer_instance_guid,
             self.request.matchdict["resource_link_id"],
         )
 
+        url_match = DOCUMENT_URL_REGEX.search(assignment.document_url)
```

Comment:
Could we call this `document_url_match`? Just to be clearer. I wasn't sure when I saw `url_match` being used below whether it referred to the `/via_url` API's URL or to the `document_url`